h2o-r/tests/testdir_algos/isofor/runit_isofor_grid_supervised.R

```diff
@@ -21,7 +21,9 @@ test.grid.resume <- function() {
       x = 1:4,
       y = 5,
       training_frame = iris.hex,
-      hyper_params = hyper_parameters
+      is_supervised = TRUE,
```

Comment:
Why do we need to set the flag here?

Reply:
I made IF unsupervised by default. This test is using the y parameter and then it should be supervised.

Reply:
I am very surprised this works - I thought IF can only have a labeled validation frame, not the training frame. Hmm.

Reply:
I didn't check the output of the grid search, since I have only used API I didn't think about that. Good point 👍

Reply:
I am trying to say that I didn't change the test, I just focused that the same code will be used with this flag. I will check what is the actual output of the grid search :)